_posts/2019/2019-06-20-managing-lets-encrypt-certificates-in-vault.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: Managing Let's Encrypt certificates in Vault
+date: 2019-06-20
+header_image: public/vault.jpg
+header_position: bottom
+header_overlay: true
+category: tech-stories
+tags: ["automation", "docker", "infrastructure"]
+authors: ["Ioannis"]
+about_authors: ["ikoutras"]
+---
+
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"} has facilitated the
+securing of websites by providing certificates free of charge. Here at ePages,
+we use them for testing our products, as well as for protecting some internal
+infrastructure. **Even though they are free, there is an extra cost: they need
+renewal (currently every 90 days at most)**. If interested in how to reduce
+this cost 😉, keep reading.
+
```

Comment:
Would love a headline for the following subsection here, as the following text block is quite long.

Reply:
Added one ("Automating certificate issuance") in 737bc2a.